scripts.js

```diff
@@ -1,24 +1,64 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
-  "use strict";
-  var result;
-
-  if (letter.className === "letter-button") {
-    result = letter.innerHTML;
-    letter.className = "letter-disabled";
-  } else {
-    result = "already picked";
-  }
-
-  console.log(result);    // for testing, remove later
-  return result;
+/*
+ * Pick from alphabet keypad. Returns the letter chosen.
+ */
+$("#alphabet-keypad").on("click", ".letter-button", pickLetter);
+
+function pickLetter() {
+  var letterChosen = $(this);
+  
+  letterChosen
+    .removeClass("letter-button")
+    .addClass("letter-disabled");
+
+  // Next two lines to be refactored into interface for
+  //   checking if the letter is in the word.
+  
+  letterChosen = letterChosen.html();
+  
+  verify(letterChosen);
 }
 
-// adding dictionary and word filter //
+// placeholder for verifying the letter
+function verify(letter) {
+  console.log(letter);
+}
 
 
+/*
+ * Hangman Object with methods loseLife() and reset()
+ */
+var hangmanObject = function () {
+  var livesLost = 0,
+      maxLife = 7;
+  
+  return {
+    loseLife : function () {
+      if (livesLost < maxLife) {
+        ++livesLost;
+        $("#hangman-frame" + livesLost).css("opacity", 1);
+      }
+    },
+    
+    isDead : function () {
+      return livesLost === maxLife;
+    },
+		
+    reset : function () {
+      for (var i = 1; i <= livesLost; ++i) {
```

Comment:
That was what I thought too, but I was afraid that for a split second all of them would be transparent and we would have a blank screen, which would look weird.